src/execution.rs

```diff
@@ -439,9 +439,13 @@ impl Print {
     fn execute(&self, exec: &mut ExecutionContext) -> Result<(), ExecutionError> {
         for value in &self.values {
             let value = value.evaluate(exec)?;
-            print!("{}", value.display_with(exec.graph));
+            if let Value::String(s) = value {
+                print!("{}", s);
+            } else {
+                print!("{}", value.display_with(exec.graph));
+            }
```

Comment:
I am not a big fan of obscuring details about the values being printed (e.g., `1` and `"1"` become indistinguishable. When debugging I prefer to clearly see what kind of value is there. But from the test I see what pattern you use for debugging, and it makes sense that you don't want the quoted version of the literals you add. I suggest changing the logic slightly, to only omit the quotes for string literals, not all string values.

```suggestion
            if let Expression::StringConstant(expr) = value {
                print!("{}", expr.value);
            } else {
                let value = value.evaluate(exec)?;
                print!("{}", value.display_with(exec.graph));
            }
```

Reply:
Ah, this is an excellent point (and suggestion).

Incidentally, would you have any objections to changing `print!` to `eprint!` in order to make the printed statements appear on `stderr` instead? The way the graph is output currently, anything output using `print` has the potential to mess up the consumers of the graph output.

Reply:
Hm. For some reason I'm not allowed to apply your suggestion here directly (GitHub complains about suggestions added to deleted lines. I'll see if I can fix it (and if not I'll just perform the change manually).

Reply:
No, using `eprint` sounds like a good idea. In the future we may want to have some flags controlling output channel and format, but for now this would make the current behavior more robust indeed.